packages/meteor-gazelle-wiki/lib/methods.js

```diff
@@ -0,0 +1,18 @@
+Wiki = {};
+
+Meteor.methods({
+  'wiki/createArticle': function (doc) {
+    var article = new Article();
+    article.set('title', doc.title);
+    article.set('body', doc.content);
+    article.set('createdBy', this.userId);
+    article.set('locked', false);
+    article.save();
+  },
+  'wiki/lockArticle': function (article) {
```

Comment:
I don't know if it's a thing currently but it should be for certain articles. 

I wouldn't make a single function just for locking articles though. From a usage perspective the user would probably need to hit edit on the article first which would then expose the locked checkbox. Hitting save would then call the article edit method.